tfutils/base.py

```diff
@@ -216,17 +242,34 @@ def __init__(self,
         self.conn = pymongo.MongoClient(host=self.host, port=self.port)
         self.conn.server_info()
         self.collfs = gridfs.GridFS(self.conn[self.dbname], self.collname)
-        recent_name = '_'.join([self.dbname, self.collname, self.exp_id, '__RECENT'])
+
+        if 'port_rec' in save_params:
```

Comment:
@chengxuz what does "port_rec" do?   @alexandonian can we clean this up to make the code more clear?